main.cpp

```diff
@@ -227,6 +454,32 @@ struct CarWash //                                   1) define an empty struct fo
     3) define waveform with adsr
  */
 
+struct Lfo       
+{
+    //has rate
+    int rate = 127; //3) member variables with relevant data types.
+    //has amount
+    int amount = 127; //3) member variables with relevant data types.
+    //has waveform
+    int waveform = 127; //3) member variables with relevant data types.
+    //has adsr
+    int adsr = 127; //3) member variables with relevant data types.
+    //has retrigger
+    int retrigger = 127; //3) member variables with relevant data types.
```

Comment:
What kind of information are these properties supposed to represent?
this is obviously just a copy/paste/rename.  Be more creative.
- [ ] revise.  pick properties that provide information and name them accordingly.

Reply:
this is the lfo with his values of my instrument